packages/toolkit/src/node/process.js

```
@@ -51,7 +51,7 @@ export const startProcess = async ({
     subprocess.stderr.on('data', data => {
       data = data.toString()
       logger(`stderr: ${data}`)
-      reject(new Error(data))
+      if (!data.includes('DeprecationWarning')) reject(new Error(data))
```

Check to prevent `setup-integration-tests` to fail silently.